source/js/buyers-guide/template-js-handler/pni-category-dropdown.js

```diff
@@ -85,6 +85,19 @@ export default () => {
   }
 
   if (dropdown) {
+    // removing styling that are exclusive used when JS is disabled or before it is loaded
+    dropdown.classList.add("tw-inline-flex");
+    dropdown.classList.remove("tw-hidden");
+    categoryWrapper.classList.add("tw-w-max", "tw-min-w-full");
+    document
+      .querySelector("#pni-category-wrapper > div:first-child")
```

Comment:
Sure I added one for this.

I do think that making unique ids names, that at the same time make sense, does get harder when we start targetting children elements, that do not really have a specific role outside of making some CSS styling easier to change. 

If you have a good id for a name in future requests, I would like you to add it as part of the request!

Reply:
That's true. Naming things is definitely one of the hard problems in programming in general. If I have an idea for an id I will add them in the future. In this case I did not really have anything because the wrapper / container name was already taken 😬